doc/src/pair_coul.rst

```diff
@@ -281,6 +288,32 @@ Streitz-Mintmire parameterization for the material being modeled.
 
 ----------
 
+.. versionadded:: TBD
```

Comment:
likewise these subsections should probably be in the same alphabetic order as the command listings at the top of the page - you can reorder the existing ones to match that and add tip

Reply:
basically for a doc page this long and complex we want to make it as easy as possible for users to find the info they want